Formula/node.rb

```diff
@@ -30,6 +30,7 @@ class Node < Formula
   depends_on "libuv"
   depends_on "openssl@3"
 
+  uses_from_macos "python3", since: :catalina
```

Comment:
```suggestion
  uses_from_macos "python", since: :catalina
```
Sorry, should've been `python`.